webapp/src/app.tsx

```diff
@@ -51,7 +51,9 @@ if (Utils.isDesktop() && Utils.isFocalboardPlugin()) {
             return
         }
 
-        history.replace(pathName.replace((window as any).frontendBaseURL, ''))
+        if (Utils.isFocalboardPlugin()) {
```

Comment:
This is not needed, this is going to be always true because it is checked in the outer if.

Reply:
Done.